### PR TITLE
[Feature][plugins] add plugin-spark-sink-redis #733

### PR DESCRIPTION
**Pull request by an-shi-chi-fan**
---

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

File: plugin-spark-sink-redis/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-sql_2.11</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.github.interestinglab.seatunnel</groupId>
+            <artifactId>plugin-spark-phoenix-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
```
Comment (wuchunfu) [inline plugin-spark-sink-redis/pom.xml:54]: replace spark version 2.11 with ${scala.binary.version}

File: plugin-spark-sink-redis/src/main/scala/io/github/interestinglab/waterdrop/spark/sink/Redis.scala [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  var redisSaveType: RedisSaveType.Value = _
+  val HASH_NAME = "redis_hash_name"
+  val SET_NAME = "redis_set_name"
+  val ZSET_NAME = "redis_zset_name"
+  val LIST_NAME = "redis_list_name"
+  val REDIS_SAVE_TYPE = "redis_save_type"
+
+
+
+
```
Comment (wuchunfu) [inline plugin-spark-sink-redis/src/main/scala/io/github/interestinglab/waterdrop/spark/sink/Redis.scala:44]: Remove useless spaces

File: plugin-spark-sink-redis/pom.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>seatunnel</artifactId>
```
Comment (simon824) [inline plugin-spark-sink-redis/pom.xml:24]: Move `artifactId` under `groupId` please.

File: plugin-spark-sink-redis/pom.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <version>2.0.4</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>plugin-spark-sink-redis</artifactId>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
```
Comment (simon824) [inline plugin-spark-sink-redis/pom.xml:35]: The properties maybe unnecessary here.

**Comment by CalvinKirs** [comment]
Hi, Is there any response?

**Comment by CalvinKirs** [comment]
Thank you for your contribution, if there is no other reason, we'd better not reopen a PR for this, he will lose the previous review. I will close this PR.

**Pull request closed without merging**